master/buildbot/test/fake/httpclientservice.py

```diff
@@ -108,8 +109,9 @@ def assertNoOutstanding(self):
                               "expected more http requests:\n {!r}".format(self._expected))
 
     def _doRequest(self, method, ep, params=None, data=None, json=None):
-        log.debug("{method} {ep} {params!r} <- {data!r}",
-                  method=method, ep=ep, params=params, data=data or json)
+        if not self.quiet:
```

Comment:
its for testing the .verbose method. You can test expectWasQuiet, because the fake is verbose.